singer_sdk/streams/rest.py

```diff
@@ -48,7 +48,7 @@ class _HTTPStream(Stream, t.Generic[_TToken], metaclass=abc.ABCMeta):  # noqa: P
     """Abstract base class for HTTP streams."""
 
     _page_size: int = DEFAULT_PAGE_SIZE
-    _requests_session: requests.Session | None
+    _requests_session: requests.Session | None = None
```

Comment:
I'd rather not add more attributes at the class level. Do you think there's a way we could update the parent `Stream` class, so that initialization doesn't fail?

Reply:
Gotcha. What do you think of https://github.com/meltano/sdk/pull/2965/commits/324c3ddfbff735b05f47d3d5993fb186e3da4880.

Reply:
Yeah, looks good to me 👍